spec/project_spec.rb

```diff
@@ -3501,28 +3501,22 @@ def callback(_args); end
         invalid_project = Optimizely::Project.new('invalid', nil, spy_logger)
         user_context = project_instance.create_user_context('user1')
         decision = invalid_project.decide(user_context, 'dummy_flag')
-        expect(decision.as_json).to include(
-          flag_key: 'dummy_flag',
-          reasons: ['Optimizely SDK not configured properly yet.']
-        )
+        expect(decision.flag_key).to eq('dummy_flag')
```

Comment:
Good to validate other decision values as well, especially "variationKey == null" and "enabled == false"

Reply:
done